ebmbot/slack.py

```diff
@@ -46,10 +51,27 @@ def notify_slack(
             sleep(1)
             error = err
 
-    logger.error(
-        "Could not notify slack",
-        channel=channel,
-        message=message_text,
-        thread_ts=thread_ts,
-        error=error,
+    msg_kwargs["text"] = COULD_NOT_NOTIFY_SLACK
```

Comment:
An additional comment (or update to the one further up) would be good to explain why we're doing this second try/except block

Reply:
Added some (hopefully well-worded) comments